src/containers/my-resources/categories-container/CategoriesContainer.constansts.tsx

```diff
@@ -0,0 +1,35 @@
+import Typography from '@mui/material/Typography'
+
+import { Categories, TableColumn, SortEnum, RemoveColumnRules } from '~/types'
+import { getFormattedDate } from '~/utils/helper-functions'
+
+import { styles } from '~/containers/my-resources/categories-container/CategoriesContainer.style'
+
+export const columns: TableColumn<Categories>[] = [
+  {
+    label: 'myResourcesPage.categories.title',
+    field: 'title',
+    calculatedCellValue: (item: Categories) => {
+      return <Typography sx={styles.title}>{item.name}</Typography>
```

Comment:
also, remove return keyword and use () instead of {}

Reply:
done